eng/native/configurecompiler.cmake

```diff
@@ -705,6 +705,11 @@ if (CLR_CMAKE_HOST_UNIX)
     endif()
   endif(CLR_CMAKE_HOST_MACCATALYST)
 
+  if(CLR_CMAKE_HOST_ARCH_ARM64 AND CLR_CMAKE_HOST_LINUX)
+    # Enable Pointer Authentication (PAC) extension
+    add_compile_options(-mbranch-protection=pac-ret)
```

Comment:
can we just pass this as part of `EXTRA_CFLAGS` or `EXTRA_CXXFLAGS` and document it? that way PAC will stay off by default.

Reply:
Done